tests/php/tests/includes/helper/test_class.wp-job-manager-com-auth-token.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+require_once JOB_MANAGER_PLUGIN_DIR . '/includes/helper/class-wp-job-manager-com-auth-token.php';
+
+/**
+ * @group helper
+ * @group helper-base
+ */
+class WP_Test_WP_Job_Manager_Com_Auth_Token extends WPJM_BaseTest {
+
+	public function testInstance_WhenCalled_ReturnSameInstance() {
+		// Arrange.
+		$instance  = WP_Job_Manager_Com_Auth_Token::instance();
+
+		// Act.
+
+		// Assert.
+		$this->assertInstanceOf( 'WP_Job_Manager_Com_Auth_Token', $instance );
+	}
+
+	public function testInstance_WhenCalled_ReturnCorrectType() {
+		// Arrange.
+		$instance  = WP_Job_Manager_Com_Auth_Token::instance();
+		$instance2 = WP_Job_Manager_Com_Auth_Token::instance();
+
+		// Act.
+
+		// Assert.
+		$this->assertSame( $instance2, $instance );
+	}
+
+	public function testGenerate_WhenPassedInvalidObjectType_ShouldReturnError() {
+		// Arrange.
+		$instance = WP_Job_Manager_Com_Auth_Token::instance();
+
+		// Act.
+		$result = $instance->generate( 'comment', 1);
+
+		// Assert.
+		$this->assertTrue( is_wp_error( $result ) );
+		$this->assertEquals( 'wpjobmanager-com-invalid-type' , $result->get_error_code() );
+	}
+
+
+	public function testGenerate_WhenPassedInvalidObjectID_ShouldReturnError() {
+		// Arrange.
+		$instance = WP_Job_Manager_Com_Auth_Token::instance();
+
+		// Act.
+		$result = $instance->generate( 'user', "test");
```

Comment:
```suggestion
		$result = $instance->generate( 'user', 'invalid_id' );
```

I suggested a format fix and I took the opportunity to just suggest another string to make the test more explicit.

Reply:
Fixed in 3e75085c236778ceb0fe8e4f055aca2b6a1323e3. Thanks!